urlparser.js

```diff
@@ -0,0 +1,30 @@
+var urlparse = 
+    function getUrlParam(key, url) {
+        if (url === undefined) {
+            url = window.location.search;
```

Comment:
It's possible this is not available in Internet explorer either: https://developer.mozilla.org/en-US/docs/Web/API/HTMLHyperlinkElementUtils/search

Reply:
will use window.location.href instead